Tasks/Common/VstsAzureRestHelpers_/VstsAzureRestHelpers_.psm1

```diff
@@ -744,19 +744,20 @@ function Get-AzureSqlDatabaseServerResourceId {
         [Object] [Parameter(Mandatory = $true)] $endpoint,
         [Object] [Parameter(Mandatory = $true)] $accessToken)
 
+    $apiVersion = "2019-06-01-preview"
```

Comment:
Can you use stable api version instead of preview?

Reply:
The last non-preview version of this api [seems to be](https://docs.microsoft.com/en-us/rest/api/sql/) from 2014 and doesn't support the 'list' operation.

Reply:
Any suggestions on a path forward here?